github-status/handler.go

```diff
@@ -148,6 +170,13 @@ func buildPublicStatusURL(status string, statusContext string, event *sdk.Event)
 	return url
 }
 
+func reportToGithub(commitStatus *sdk.CommitStatus, event *sdk.Event) error {
+	if os.Getenv("use_checks") == "false" {
```

Comment:
Thank you for giving us a way to feature flag this.

So by default is this on?

Reply:
On by default, since we are not degrading any functionality, only adding a bit. Can be false by default, if you'd prefer.